MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/javaservices/DirectionsActivity.java

```diff
@@ -74,7 +75,7 @@ protected void onCreate(Bundle savedInstanceState) {
     mapView.getMapAsync(new OnMapReadyCallback() {
       @Override
       public void onMapReady(@NonNull MapboxMap mapboxMap) {
-
+        DirectionsActivity.this.mapboxMap = mapboxMap;
```

Comment:
Shall we directly pass the `mapboxMap` to the `getRoute` method as an argument? Since there seems to be only one usage of the `mapboxMap` in this activity.

Reply:
Sure. Fixed in my latest commit